src/character.h

```diff
@@ -1550,6 +1550,7 @@ class Character : public Creature, public visitable<Character>
         void drench( int saturation, const body_part_set &flags, bool ignore_waterproof );
         /** Recalculates morale penalty/bonus from wetness based on mutations, equipment and temperature */
         void apply_wetness_morale( int temperature );
+        int Character::return_heartrate() const;
```

Comment:
i'd probably just call this `heartrate`, there's really no reason to have the word `return` in the function name because it doesn't convey any information. alternatives are `heart_rate`, `heartrate_bpm`, `heart_rate_bpm`
it is bpm right?

Reply:
"bpm" is right. I will use "heartrate_bpm"

Reply:
Also dont need the character:: in the .h file, as its already in the character namespace there.